go/filemanager/pfsmodules/cp.go

```diff
@@ -0,0 +1,62 @@
+package pfsmodules
+
+import (
+	"flag"
+	"fmt"
+	"strconv"
+)
+
+const (
+	cpCmdName = "cp"
+)
+
+// CpCmdResult means the copy-command's result.
+type CpCmdResult struct {
+	Src string `json:"Path"`
+	Dst string `json:"Dst"`
+}
+
+// CpCmd means copy-command.
+type CpCmd struct {
+	Method string
+	V      bool
+	Src    []string
+	Dst    string
+}
+
+// NewCpCmdFromFlag returns a new CpCmd from parsed flags.
+func NewCpCmdFromFlag(f *flag.FlagSet) *CpCmd {
+	cmd := CpCmd{}
+
+	cmd.Method = cpCmdName
+	cmd.Src = make([]string, 0, f.NArg())
+
+	var err error
+	f.Visit(func(flag *flag.Flag) {
+		if flag.Name == "v" {
+			cmd.V, err = strconv.ParseBool(flag.Value.String())
+			if err != nil {
+				panic(err)
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#dont-panic

I think in general no code should panic besides maybe it's ok if code in package `main`.